tests/dummy/app/index/template.hbs

```diff
@@ -18,28 +18,27 @@
     <section id="installation">
       <h2>Installation</h2>
       <div class="grid">
-        {{#window-pane}}
+        <WindowPane>
           <span class="ps1">&gt; </span>ember install ember-page-title
-        {{/window-pane}}
+        </WindowPane>
       </div>
     </section>
 
     <section id="playground">
       <h2>Playground</h2>
       <div class="grid">
-        {{#page-title-pane as |p|}}
-          {{#p.matryoshka items=model.tokens as |token|}}
+        <PageTitlePane as |p|>
+          <p.matryoshka @items={{this.model.tokens}} as |token|>
```

Comment:
`@model` become available in Ember.js v3.14+ and we are targeting v3.12, so has to be `this.model` for now.

Reply:
Actually I think the latest ember-cli-update drops 3.12, but I'm unsure if the title RFC says 3.12+ or not.

Reply:
RFC does not mention version. What I meant here is that current Travis setup in this repo targets v3.12.

Reply:
Maybe we should run ec-update again to get the latest support matrix.

Reply:
We have all LTS versions from 3.12 to 3.20 included.

Reply:
3.12 was dropped in the latest new-addon output: https://github.com/ember-cli/ember-addon-output/tree/v3.21.2 It's now 3.16 +

Reply:
Yeah, sure. I would do the 3.12 drop in an another PR. Bumping dependencies and so on.